markdown/guide/linking-and-external-links.md

```diff
@@ -83,7 +83,7 @@ _Note that the Engine name, which is normally dasherized, is camel-cased here in
 You can use these external routes within your Engine via the `{{link-to-external}}` component:
 
 ```hbs
-{{link-to-external "home"}}
+{{link-to-external "home" "home"}}
```

Comment:
Can we use "Go Home" (or something similar) as the second parameter? That way it will be more clear what each argument is doing.